logger/mask.go

```diff
@@ -114,7 +127,11 @@ func findTagAndMask(doc *xmlquery.Node, maskChar string, toMask MaskData) {
 //				"first": "1st!!!!!!!!!!!!!!!d",
 //				"second": "2nd of second third",
 //				"third": "3rd of second third",
-//			}
+//			},
+//			"fourth": [
+//				{ "email": { "value": "f!!!!!!@!!!!!!com" } },
+//				{ "email": { "value": "s!!!!!!@!!!!!!com" } }
+//			]
 //		}
 //	}
```

Comment:
for large comment block I think it is cleaner to use this comment format 
/*
...
*/

Reply:
Done in f4c8916. Also tried to make it readable on Go doc using https://github.com/golang/pkgsite:

![image](https://github.com/user-attachments/assets/55c265e2-f6b7-42ca-b6e4-f60de5443f7c)